src/index.html

```diff
@@ -15,18 +15,13 @@ <h2 class="recent">Recent scores</h2>
             <a href="#" class="refresh">Refresh</a>
           </div>
           <ul class="high-list">
-            <li class="high-score">John : 100</li>
-            <li class="high-score">Jane : 95</li>
-            <li class="high-score">Luke : 90</li>
-            <li class="high-score">Ninja boy : 87</li>
-            <li class="high-score">Sarah : 85</li>
           </ul>
         </div>
         <div class="add-score">
           <h2 class="add">Add score</h2>
           <form action="#">
             <input type="text" class="name" placeholder="Your name" />
-            <input type="text" class="name" placeholder="Your score" />
+            <input type="text" class="name" id="scored" placeholder="Your score" />
```

Comment:
- [ ] [Optional] Large numbers in score can break your app, please consider adding some kind of validation to your form

![Screenshot_20220513_050332](https://user-images.githubusercontent.com/25227022/168209216-e4a3ce25-c0e7-4b6b-acbb-2dd09c1f30e1.png)